animal-sniffer-maven-plugin/src/it/jdk14-with-sig14-from-cli/invoker.properties

```diff
@@ -1 +1 @@
-invoker.goals=org.codehaus.mojo:animal-sniffer-maven-plugin:1.15-SNAPSHOT:check -Danimal.sniffer.signature=org.codehaus.mojo.signature:java14:1.0
+invoker.goals=org.codehaus.mojo:animal-sniffer-maven-plugin:1.16-SNAPSHOT:check -Danimal.sniffer.signature=org.codehaus.mojo.signature:java14:1.0
```

Comment:
Does not look like it belongs.

This file ought to be running simply `check` (or delete and specify everything in the POM). But apparently unrelated to this PR.

And it looks like you are missing `src/it/github-22/pom.xml`.